.gitignore

```diff
@@ -6,3 +6,6 @@ tmp
 
 **/*.pos
 **/test_output/
+
+# IDEs
+.idea/
```

Comment:
This should go to your global `~/.gitignore`

Reply:
Yes, thx.